app/routes/admin.qb-streaming.$id.tsx

```diff
@@ -57,13 +57,21 @@ export const action = async ({ params, request }: ActionFunctionArgs) => {
       );
       if (!nflGame) throw new Error(`Game not found for QB`);
 
-      await createQBStreamingWeekOption({
-        playerId,
-        isDeep: !!isDeep,
-        pointsScored: 0,
-        qbStreamingWeekId,
-        nflGameId: nflGame.id,
-      });
+      const uniqueId = `${qbStreamingWeek.year}-${qbStreamingWeek.week}-${player.id}`;
```

Comment:
Do we need to actually save a unique ID in the database for this? Can we just check that the player ID doesn't already exist in the year and week?